akka-stream/src/main/scala/akka/stream/impl/fusing/StreamOfStreams.scala

```diff
@@ -334,8 +334,9 @@ import scala.collection.JavaConverters._
     override protected def onTimer(timerKey: Any): Unit = {
```

Comment:
should we also do a `closedSubstreams = Collections.unmodifiable(Set.empty)` in the `preStart` so no one accidentally uses it?

Reply:
Why not add it in the initialization directly? Any concerns?

Reply:
Yeah thats good too